start.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python3
```

Comment:
Will this file get served to users once the PR is checked in? It seems like pages by default serves all files present--is there a way we can blacklist this file?

Reply:
Really good question.  After reviewing GitHub Pages documentation, it does not look like there is a way to blacklist files.  However, this is an open repository anyway, and since the file isn't linked to any resource a user would have to explicitly navigate to it.  Furthermore, downloaded scripts do not have the executable bit enabled; that has to be set manually.  Finally, even if a user did download and execute the script, all they'd get is a self-signed cert and a simple web server.  This script doesn't change any certificate trust or store settings on the system, and I can't really see a way to use it for malicious intent.  Do you have any particular concerns?

Reply:
I think you outlined everything. I think it's worth noting these details at a high level in the Python file (just so that future maintainers aren't surprised that it can be accessed via the web browser, and that that's okay).